src/js/subtraction/components/Toolbar.jsx

```diff
@@ -1,32 +1,50 @@
-import React from "react";
+import React, { useState } from "react";
 import { connect } from "react-redux";
 import { checkAdminRoleOrPermission } from "../../administration/utils";
 import { InputSearch, LinkButton, Toolbar } from "../../base";
+import { useUrlSearchParams } from "../../utils/hooks";
 import { findSubtractions } from "../actions";
 
-export const SubtractionToolbar = ({ term, onFind, canModify }) => {
-    let createButton;
+/**
+ * Managing the toolbar search filtering
+ *
+ * @param onFind - The dispatch action to update the search filtering results
+ * @param canModify - The user's permissions to modify
+ * @returns Toolbar - A search filtering toolbar
+ */
+export const SubtractionToolbar = ({ onFind, canModify }) => {
+    const [value, setValue] = useUrlSearchParams("find");
```

Comment:
```suggestion
    const [term, setTerm] = useUrlSearchParams("find");
```
more descriptive imo